package.json

```diff
@@ -12,28 +12,33 @@
     "build": "vite build -m production",
     "build:lch": "LCH=1 vite build -m production && ./lch.sh",
     "lint": "tsc --noEmit && eslint . && stylelint **/*.css",
-    "test": "pnpm audit --prod --audit-level high && pnpm lint && pnpm build && size-limit && LCH=1 pnpm build"
+    "test": "pnpm audit --prod --audit-level high && pnpm lint && pnpm build && size-limit && LCH=1 pnpm build",
+    "build:shapes": "LCH= npx ts-node tools/color-shapes/index.ts && LCH=1 npx ts-node tools/color-shapes/index.ts"
```

Comment:
Now new developer after `git pull …` and `pnpm install && pnpm test` will have an issue.

Let’s add it to `build` and `build:lch` if it is fast.

Reply:
And put generated files to `.gitignore` (all generated files, especially so big one, should not be in repository)

Reply:
ok, but it will slow down pipeline for about 1-1.5 minutes

Reply:
On CI it is OK. At least we will always be sure that generators work (and wasn’t broke on some `pnpm update`).